app/ante/utils_test.go

```diff
@@ -343,6 +345,52 @@ func (suite *AnteTestSuite) CreateTestEIP712MsgSubmitEvidence(from sdk.AccAddres
 	return suite.CreateTestEIP712CosmosTxBuilder(from, priv, chainId, gas, gasAmount, msgEvidence)
 }
 
+func (suite *AnteTestSuite) CreateTestEIP712SubmitProposalV1(from sdk.AccAddress, priv cryptotypes.PrivKey, chainId string, gas uint64, gasAmount sdk.Coins) client.TxBuilder {
+	// Build V1 proposal messages. Must all be same-type, since EIP-712
+	// does not support arrays of variable type.
+	authAcc := suite.app.GovKeeper.GetGovernanceAccount(suite.ctx)
+
+	proposal1, ok := types5.ContentFromProposalType("My proposal 1", "My description 1", types5.ProposalTypeText)
+	suite.Require().True(ok)
+	content1, err := govtypes.NewLegacyContent(
+		proposal1,
+		sdk.MustBech32ifyAddressBytes(sdk.GetConfig().GetBech32AccountAddrPrefix(), authAcc.GetAddress().Bytes()),
```

Comment:
why do you need to use the config? is it using another prefix? If so we should change it in the setup so that we don't need to call this

Reply:
This is used because `NewLegacyContent(...)` expects the Governance module's address as a Bech32 string. I was under the impression that this is the standard implementation to convert address bytes to a Bech32 address string.

Reply:
This is used because NewLegacyContent(...) expects the Governance module's address as a Bech32 string. Is there a better way to do this?